src/main/webapp/app/view/project/admin/responsibilities/project-responsibilities.component.html

```diff
@@ -35,12 +35,16 @@ <h3 id="page-heading">
                 <th scope="col">
                     <span jhiTranslate="eventPlannerApp.responsibility.name">Name</span>
                 </th>
+                <th scope="col">
+                    <span jhiTranslate="eventPlannerApp.responsibility.color">Color</span>
+                </th>
                 <th scope="col"></th>
             </tr>
             </thead>
             <tbody>
             <tr *ngFor="let responsibility of responsibilities ;trackBy: trackId">
                 <td>{{ responsibility.name }}</td>
+                <td><div class="btn color-preview" [ngStyle]="{'background-color':responsibility.color? responsibility.color: defaultColor}" ></div></td>
```

Comment:
Würde wohl nicht die Klasse `btn` verwenden. So kann man das Farbkästchen anklicken und es verändert sich bisschen bei der Farbe.

Reply:
ah, dachte ich habe den klick via css (`.color-preview`) disabled. ich finde den `btn` nice wegen den runden ecken.

Reply:
also bei mir weder im Firefox, noch in Chrome klickbar.. 🚀